cli/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "immich-cli",
+  "dependencies": {
+    "axios": "^1.4.0",
+    "form-data": "^4.0.0",
+    "mime-types": "^2.1.35",
+    "systeminformation": "^5.18.4"
```

Comment:
I don't think we need this dependency.

Reply:
Hm...I tried removing systeminformation (was that the one you meant?) and it didn't compile. Don't know what it does, t though

Reply:
Looks like we get some sort of identifier with it, maybe it's fine for now.